vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.jsp

```diff
@@ -44,6 +44,97 @@ function uploadImage() {
 } 
 	
 $(document).ready(function() {
+	//-------- edit description --------
+	$("#submitDescription").hide()
+	$("#editDescription").click(function() {
+		var val = $("#description").val();
+		$("#description").html(function (i, val) {
```

Comment:
why do you pass a function here and not just the string?

Reply:
by passing a function to HTML it allows me to access the old val of the HTML I am replacing. It essentially saves me a line or two of code.

http://api.jquery.com/html/#html-function